core/extension/gdextension.cpp

```diff
@@ -434,6 +473,9 @@ void GDExtension::_register_extension_class_internal(GDExtensionClassLibraryPtr
 	extension->gdextension.is_virtual = p_extension_funcs->is_virtual;
 	extension->gdextension.is_abstract = p_extension_funcs->is_abstract;
 	extension->gdextension.is_exposed = p_extension_funcs->is_exposed;
+#ifdef TOOLS_ENABLED
+	extension->gdextension.is_runtime = is_runtime;
```

Comment:
> Do we really need to but this assignment in a conditional block?

Well, I tried to make it so that this feature only exists in the editor and doesn't add anything to the export templates. I don't know that it _needs_ to be done that way, but I personally prefer it, because the compiler will help us avoid anything leaking into the export templates (ie. if some non-editor code tries to access `is_runtime`, it'll be a compile-time error, because that property doesn't exist).

> Also, is the default value defined?

I'm not sure what you mean by this? The `p_extension_funcs` which is passed to us from the GDExtension needs to have all the fields on the struct filled out, and earlier in this function there's `bool is_runtime = (bool)p_extension_funcs->is_runtime`, so this value will definitely be set.